apis/validator/duties/proposer.v2.yaml

```diff
@@ -0,0 +1,70 @@
+get:
+  tags:
+    - ValidatorRequiredApi
+    - Validator
+  summary: "Get block proposers duties"
+  operationId: "getProposerDutiesV2"
+  description: "Request beacon node to provide all validators that are scheduled to propose a block in the given epoch.
+
+    Duties should only need to be checked once per epoch,
+    however a chain reorganization could occur that results in a change of duties. For full safety,
+    you should monitor head events and confirm the dependent root in this response matches. After Fulu,
+    different checks need to be performed as the dependent root changes due to deterministic proposer lookahead.
+
+
+    Before Fulu:
```

Comment:
should explicitly state we're required from gloas...

Reply:
its marked required, and not required in fulu, thats the wierdness im trying to encapsulate...

Reply:
do you mean because it's tagged as `ValidatorRequiredApi`? we can remove that tag for now and not deprecate the v1 api if that makes it clearer, and then later add the tag again+ deprecate v1

Reply:
or we add the same note I added in the PR description and later remove it again, both works for me

Reply:
Given the version for fulu has shipped i guess its not a deal breaker either way, it just might be clearer since it's "validatorRequired" that we're fairly explicit... Maybe removing that tag for now is 'easy' - i'd be happy with that... 
I was just thinking a `Note: not required until GLOAS` would be an easy way to be explicit...